PhotoKeeper/PhotoGridViewController.swift

```diff
@@ -7,114 +7,118 @@
 //
 
 import UIKit
+import RealmSwift
 
 class PhotoGridViewController: UICollectionViewController, UICollectionViewDelegateFlowLayout {
     var store = PhotoStore.shared
-    var images = [Photo]()
+    var photos: Results<PhotoObject>!
     var topBarHeight: CGFloat = 0
-    
-    private let reuseIdentifier = "PhotoCell"
+
+    let reuseIdentifier = "PhotoCell"
     let black = UIColor(hue: 0.5278, saturation: 0.9, brightness: 0.23, alpha: 1.0)
-    let INSET_SIZE: CGFloat = 5
-    let documentsDirectory = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
-    
+    let INSET: CGFloat = 5
```

Comment:
a better way to handle constants that live in a class is to define a struct for those variables like:

```swift
private struct Layout {
  static let inset: CGFloat = 5
}

// usage:

Layout.inset
```

also, there is actually a type for defining insets: `UIEdgeInsets` which would be like this:

```swift
private struct Layout {
  static let insets = UIEdgeInsets(top: 10, left: 5, bottom: 5, right: 10)
}

// usage:

Layout.insets.left
```